docs/spfx/use-aadhttpclient.md

```diff
@@ -188,12 +188,12 @@ Global and SharePoint administrators can also use the SharePoint Online Manageme
   > [!NOTE]
   > SharePoint doesn't verify if the requested permissions have already been granted or not, so before approving or rejecting a permission request, check which permissions have already been granted in your tenant.
 
-- To *approve the specific permission request*, use the **Approve-SPOTenantServicePrincipalPermissionRequest -RequestId <Guid>** cmdlet, specifying the ID of the permission request that you want to approve.
+- To *approve the specific permission request*, use the **Approve-SPOTenantServicePrincipalPermissionRequest -RequestId &lt;Guid&gt;** cmdlet, specifying the ID of the permission request that you want to approve.
 
     > [!NOTE]
     > If you try to approve a request for a permission that has already been granted, you get an error.
 
-- To *deny a permission request* (if the requested permission has already been granted, or the request is against your organizational policies), use the **Deny-SPOTenantServicePrincipalPermissionRequest -RequestId <Guid>** cmdlet, specifying the ID of the permission request that you want to deny.
+- To *deny a permission request* (if the requested permission has already been granted, or the request is against your organizational policies), use the **Deny-SPOTenantServicePrincipalPermissionRequest -RequestId %lt;Guid&gt;** cmdlet, specifying the ID of the permission request that you want to deny.

```

Comment:
Correct markdown escaping of angled brackets should be prefixed with`\`; don't change to the HTML encoding.